index.js

```diff
@@ -214,6 +214,14 @@ function showSearch() {
     });
 }
 
+/*
+ * Opens the bookmark menu
+ */
+function showBookmarks() {
+    let activeWindow = windowUtils.getMostRecentBrowserWindow();
+    UITour.showMenu(activeWindow, 'bookmarks');
```

Comment:
I think I'll just throw a try catch. Much less likely to break anything else since it is only for this one call + I can add this to our cleaning javascript bug to make the code more DRY